### ISSUE TITLE
General usability

### ISSUE THREAD

**Issue by smellyd** [FreeCodeCamp/FreeCodeCamp#2420]
Challenge http://www.freecodecamp.com/challenges/waypoint-clone-an-element-using-jquery has an issue. Please describe how to reproduce it, and include links to screenshots if possible.

This is a general "issue". When editing the code the instructions are constantly moving. It makes it hard to follow as your editing. Obviously workable, just annoying.

Thanks for all of your hard work! This is a great program!

**Comment by BerkeleyTrue** [comment]
Closing as duplicate #2137
